packages/pluggableWidgets/line-chart-native/src/LineChart.editorConfig.ts

```diff
@@ -36,23 +36,23 @@ export function check(values: LineChartPreviewProps): Problem[] {
             // @ts-ignore
             if (lines.staticDataSource.type === "null") {
                 errors.push({
-                    property: "staticDataSource",
+                    property: `lines/${index + 1}/staticDataSource`,
                     severity: "error",
                     message: `No data source configured for static line located at position ${index + 1}.`
```

Comment:
This index info is probably not necessary anymore. Same for bar-chart-native.

Reply:
It is required!

Reply:
Although verbosity doesn't hurt :)

Reply:
I would agree if our consistency checks was clear enough to determine where is the error. 
![image](https://user-images.githubusercontent.com/7356299/112347581-46d9ee00-8cc7-11eb-871d-b5801e2522b7.png)
 We should keep them as is IMO